modules/persistent-storage-byok.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-azure.adoc
+// storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+// storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_content-type: CONCEPT
+[id="byok_{context}"]
+= About "bring your own key" encryption
+
+// TP or GA???
+
+The "Bring your own key" (BYOK) feature encrypts root volumes and the default storage class.
+
+When starting up managed {product-title} clusters, you can pass a custom Key Management Service (KMS) key to the installer, and the installer and `cluster-storage-operator` encrypts the root volumes for the nodes in the cluster, and also applies this key to encrypt the default storage class.
+This allows you to encrypt your owned volumes with your selected key, instead of the platform's default account key.
+
+This features supports the following storage types:
+
+* Amazon Web Services (AWS) Elastic Block storage (EBS)
+
+* Microsoft Azure Disk storage
+
+* Google Cloud Platform (GCP) persistent disk (PD) storage
+
+// Procedure on how to make this happen???
```

Comment:
I wrote a procedure for this for Azure:
https://docs.openshift.com/container-platform/4.12/installing/installing_azure/enabling-user-managed-encryption-azure.html
With an accompanying module inside the installation assemblies for finalizing:
https://docs.openshift.com/container-platform/4.12/installing/installing_azure/installing-azure-network-customizations.html#finalizing-encryption_installing-azure-network-customizations

Not sure about AWS and GCP though. @jhixson74 Do you know if we need to update the "Enabling user managed encryption" page with any new procedure or configuration to describe this feature?

Reply:
I also have https://issues.redhat.com/browse/CORS-2080 which I believe is covering the installer portion of extending the azure disk encryption to the default storage class